src/libs/shim-interface/src/lib.rs

```diff
@@ -25,6 +25,41 @@ use kata_types::config::KATA_PATH;
 
 pub const SHIM_MGMT_SOCK_NAME: &str = "shim-monitor.sock";
 
+fn get_uds_with_sid(short_id: &str, kata_run_path: &str) -> Result<String> {
+    // /kata/run/62db246d6c4ffc4a20b56b7259a0346a78aeccb46c4bdc3ef97899d787fd5797
+    let p = Path::new(kata_run_path)
+        .join(short_id)
+        .join(SHIM_MGMT_SOCK_NAME);
+    if p.exists() {
+        return Ok(format!("unix://{}", p.display()));
```

Comment:
I think `.display()` is unnecessary here

Reply:
Thx @Tim-Zhang I will fix it later.